exercises/concept/rpn-calculator/.docs/introduction.md

```diff
@@ -0,0 +1,6 @@
+# Introduction
+
+Rust's vector implementation, [`std::vec::Vec`](https://doc.rust-lang.org/std/vec/struct.Vec.html), can be used to simulate a stack by using its `push()` and `pop()` methods.
```

Comment:
```suggestion
Rust's vector implementation, [`std::vec::Vec`](https://doc.rust-lang.org/std/vec/struct.Vec.html), can be used as a stack by using its `push()` and `pop()` methods.
```

It's not really simulating it; it's really a stack.

Reply:
Fixed in 7f6a9a4